include/OpenSpaceToolkit/Astrodynamics/Trajectory/CoordinatesSubsets/Invariant.hpp

```diff
@@ -0,0 +1,54 @@
+/// Apache License 2.0
+
+#ifndef __OpenSpaceToolkit_Astrodynamics_Trajectory_CoordinatesSubsets_Invariant__
+#define __OpenSpaceToolkit_Astrodynamics_Trajectory_CoordinatesSubsets_Invariant__
+
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/CoordinatesBroker.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace trajectory
+{
+namespace coordinatessubsets
+{
+
+using ostk::core::types::String;
+
+using ostk::physics::time::Instant;
+
+using ostk::astro::trajectory::CoordinatesSubset;
+using ostk::astro::trajectory::CoordinatesBroker;
+
+/// @brief  Invariant (frame-indepdentent) coordinate subset.
+
+class Invariant : public CoordinatesSubset
```

Comment:
```suggestion
class FrameInvariant : public CoordinatesSubset
```

Should we make it more explicit?